craigslist-apartments.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let axios = require('axios');
+let cheerio = require('cheerio');
+let filters = require('./filters');
+
+const Filter = new filters();
+let mountedURL = mountURL(Filter); 
+
+function mountURL(filter) {
+ let url = `https://vancouver.craigslist.org/search/apa?availabilityMode=0&bundleDuplicates=1&hasPic=1`;
+ if(filter.getMaxPrice()) {
+   url = url + `&max_price=${filter.getMaxPrice()}`;
+ }
+ if(filter.getMinPrice()) {
+   url = url + `&min_price=${filter.getMinPrice()}`; 
+ }
+ if(filter.getKmFromPostalCode()) {
+   url = url + `&search_distance=${filter.getKmFromPostalCode()}`; 
+ }
+ if(filter.getPostalCode()) {
+   url = url + `&postal=${filter.getPostalCode()}`; 
+ }
+ if(filter.getMinSqft()) {
+   url = url + `&minSqft=${filter.getMinSqft()}`; 
+ }
+ return url;
+}
+
+function getListOfRentals(totalLastSearch, listOfRentals) {
```

Comment:
You should receive a "filter" object as part of the params.

Reply:
I have created a new function where I am passing the filter.